### PR TITLE
Fix dashboard checkbox filters and left margin

### LINKED ISSUE #5895: Dropdown checkbox filters on dashboard
I noticed on the dashboard the dropdown checkboxes to filter which updates should show doesn't work. I can PR to fix this if you like.
filter-dropdown

### PR DESCRIPTION
**Pull request by michelelong**
Fixes #5895 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!
fix-filter1

### CODE REVIEW

File: app/views/dashboard/_activity.html.erb [hunk 1]
```diff
@@ -20,54 +20,54 @@
...
+      <li class="form-check dropdown-item">
         <label class="form-check-label">
-          <input data-type="comment" class="node-type node-type-comment" type="checkbox" checked="checked" /> 
+          <input data-type="comment" class="node-type node-type-comment form-check-input" type="checkbox" checked="checked" /> 
           <%= t('dashboard._activity.dropdown.comments') %>
         </label>
-      </a>
+      </li>
 
-      <a class="d-lg-none dropdown-item" href="#">
```
Comment (michelelong) [inline app/views/dashboard/_activity.html.erb]: The wiki link is set to hide on large screens. I don't know why so I left it as it was.

**Comment by michelelong** [comment]
@publiclab/reviewers

File: app/views/dashboard/_activity.html.erb (hunk 1 above)
Comment (gauravano) [reply app/views/dashboard/_activity.html.erb]: hey @michelelong, on larger screens, you'll notice, we show a sidebar which has wiki tab in it and that shows all the updates of wiki edits. But, in mobile view or in smaller screens, we can't have a sidebar so we show wiki edit updates here too.

Comment (michelelong) [reply app/views/dashboard/_activity.html.erb]: I see now, thanks. Updated the wiki checkbox classes for mobile.

**Comment by michelelong** [comment]
Yikes! Sorry about the mess of commits. I tried to squash them into one, but it just sends me into a loop of committing the same thing.

**Comment by michelelong** [comment]
> How about putting a test for this?
I'm not really familiar with Ruby, but I can look at some of the other tests and give it a try.

**Comment by michelelong** [comment]
I'm having a issue with the Bootstrap classes styling. 
With the form-check class on the wiki link everything looks good, but it shows up large screens even though d-lg-none is there.
`<li class="d-lg-none form-check dropdown-item">`
When I remove the form-check class, the wiki link doesn't show up on large screens as intended, but then the alignment on mobile is slightly off.
`<li class="d-lg-none dropdown-item">`
spacing-issue-wiki

**Comment by michelelong** [comment]
Ok, it's finally working and looking correct. I also took a crack at writing tests to show the filtering is working. Should I commit it with this, separately or not all because I don't know what I am doing?
One of the tests
```
test "filter_notes_on_dashboard" do
    within("ul.dropdown-menu") do
      uncheck("Research notes")
    end
    assert_select('div > .note-container-note', {count: 0})
    take_screenshot
  end
```

**Comment by SidharthBansal** [comment]
Hi, are you still stuck in this issue?

**Comment by jywarren** [comment]
Hi @SidharthBansal I believe michelelong said they'd be unable to complete it, so we can close it but ask anyone interested in solving the original issue to build off of this excellent start in submitting a new solution? Thanks!

**Comment by jywarren** [comment]
However, in this case, i believe the dashboard will be rebuild soon under the new style guide, so we can probably close this as it'll be addressed there. Thanks a lot, @michelelong!

**Pull request closed without merging**